### DISCUSSION TITLE
Possible to control the max height of Content inside of Select, when not using popper?

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [radix-ui/primitives#2052] [Help] [answered]
If you have a long list, like a list of countries for example, is it possible to set a maximum height for the `Content` that pops up? Currently it extents the full height of the viewport.

Conversely, if I set `Content`'s `position` to `"popper"`, I can control the height, but I'd like to have the popup be aware of the edges of the screen and change position accordingly, so popper doesn't seem like the right option.

Cheers

**Comment by joaom00** [comment, marked as answer]
I'm afraid this is not possible due to the fact of the complexity of the `item-aligned` position

**Comment by joaom00** [comment]
>  but I'd like to have the popup be aware of the edges of the screen and change position accordingly, so popper doesn't seem like the right option.

Are you saying the `avoidCollisions`? Isn't working?
